WordPress/src/main/res/values/strings.xml

```diff
@@ -893,6 +893,7 @@
 
     <string name="stats_posting_activity_action">navigate all stats for this period</string>
     <string name="stats_posting_activity_content_description">"The days with %1$s views for %2$s are : %2$s %3$s, tap for more."</string>
+    <string name="stats_posting_activity_label_content_description">"Posting Activity for %1$s</string>
```

Comment:
The string starts with `"`, is it intentional?

Reply:
Resolved b5f6951